src/main/app/src/app/shared/material-form-builder/form/form/form.component.html

```diff
@@ -23,13 +23,15 @@
         </button>
         <button mat-flat-button color="primary"
                 [disabled]="formGroup.invalid"
+                id="opslaan_button"
```

Comment:
Deze id's moeten dynamisch bepaald worden omdat er niet met zekerheid gezegd kan worden of ze maar 1 keer voorkomen